application/common/components/passwordStore/Ldap.php

```diff
@@ -312,9 +313,9 @@ protected function updatePassword($user, $password)
     }
 
     /**
-     * @param \Adldap\Models\Entry $user
+     * @param \LdapRecord\Models\OpenLDAP\Entry $user
```

Comment:
I prefer to put 'use' statements above to simplify this. Perhaps a use this as LDAP_Entry?